src/cpp/TextNoteCollection.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * TextNoteCollection.h
+ *
+ *  Created on: 10 Sep 2022
+ *      Author: MouseCreator
+ */
+
+#ifndef TEXTNOTECOLLECTION_H_
+#define TEXTNOTECOLLECTION_H_
+#include <vector>
```

Comment:
Include order:
1. something.h in something.cpp
2. other headers from same project
3. headers from 3rd-party libraries
4. headers from standard C++ library
5. headers from standard C library